client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -258,14 +258,15 @@ export const isCustomsFormRequired = createSelector(
 				'An origin address phone number is required for this shipment.'
 			);
 		} else if (
+			'US' === country &&
 			10 !==
 			phone
 				.split( /\D+/g )
 				.join( '' )
 				.replace( /^1/, '' ).length
 		) {
 			errors.phone = translate(
-				'Customs forms require a 10-digit phone number. ' +
+				'Origin address require a 10-digit phone number. ' +
```

Comment:
Do we hijack the customs flow to validate the origin address?

```suggestion
				'Origin phone number has to be 10-digit. ' +
```

Reply:
Before this changes, origin phone number only required if customs form is also required. Now since origin phone number is always required I thought we could change the `errors.phone` text here with origin address related message.